sys/cddl/contrib/opensolaris/uts/common/sys/dtrace.h

```diff
@@ -740,14 +740,22 @@ typedef struct dof_sec {
 	((x) == DOF_SECT_XLEXPORT) ||  ((x) == DOF_SECT_PREXPORT) || 	\
 	((x) == DOF_SECT_PRENOFFS))
 
+#if __has_feature(capabilities)
+typedef uintcap_t dtrace_uarg_t;
+#else
+typedef __uint128_t dtrace_uarg_t;
```

Comment:
Why does this need to be a uint64? Does this not break ABI for non-purecap?

Reply:
I'll do my best to fix this, but I'm not sure it possible in a nice way..

Reply:
Fixed :)